CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLWorker.cc

```diff
@@ -250,98 +256,88 @@ SiPixelLorentzAnglePCLWorker::SiPixelLorentzAnglePCLWorker(const edm::ParameterS
   max_depth_ = 400.;
   min_drift_ = -1000.;  //-200.;(iConfig.getParameter<double>("residualMax"))
   max_drift_ = 1000.;   //400.;
-
-  for (int lay = 0; lay < 4; ++lay) {
-    for (int lad = 0; lad < 64; ++lad) {
-      for (int mod = 0; mod < 8; ++mod) {
-        newmodule[lay][lad][mod] = false;
-      }
-    }
-  }
-  newmodule[0][3][6] = true;
-  newmodule[0][5][6] = true;
-  newmodule[0][3][3] = true;
-  newmodule[0][3][1] = true;
-  newmodule[0][5][3] = true;
-  newmodule[0][7][3] = true;
+  ypitch_ = 0.0150;
```

Comment:
@mmusich isnt there a way to get the pixel size from the geometry instead of having it in a hardcode like this?

Reply:
Yes

Reply:
E.g. from here: https://github.com/cms-sw/cmssw/blob/49cd0eb9a47e85d38023c5c158e60be18687ee04/Geometry/CommonTopologies/interface/PixelTopology.h#L28 passing from the geomdets

Reply:
Here's a fully worked out example https://cmssdt.cern.ch/dxr/CMSSW/source/SimTracker/SiPixelDigitizer/test/PixelDigisTest.cc#462

Reply:
Thanks @mmusich , please go ahead with this version @wweiphy